.github/workflows/ci.yml

```diff
@@ -519,6 +551,7 @@ jobs:
       - tsan
       - riscv32
       - riscv64
+      - android_arm64
```

Comment:
nit: maybe alphabetize platforms, or put android_* before riscv_* (since we do more benchmarking on Android / have more developers using it)

same for the order in the file above this

---

if comments are okay, we could also label the different build categories, e.g.
```yaml
  summary:
    runs-on: ubuntu-20.04
    needs:
      # basic builds
      - build_runtime
      - test_runtime
      - build_all
      - build_test_all_bazel
      - test_all
      - test_gpu
      - host_tools_assertions
      # tensorflow
      - test_tf_integrations
      - test_tf_integrations_gpu
      # special configurations
      - asan
      - tsan
      # platforms
      - riscv32
      - riscv64
      - android_arm64
```

Reply:
If we comment here then I think we should have fenced comments throughout the file. We also might end up with something that breaks the topological sorting. I'm down to try some organization though

Reply:
Yeah, just something to keep in mind as this list stabilizes. I do still think that Android should go before riscv (at least based on alphabetical ordering)

Reply:
Ok then the first question is going to be "where do we put `build_benchmarks`", which I'm introducing in the next PR? It requires the TF integrations and full build, but doesn't really fit in any of the categories IMO

Reply:
Rearranged